django-server/server/api/urls.py

```diff
@@ -6,5 +6,7 @@
 router.register(r'cafeteria', views.CafeteriaViewSet)
 
 urlpatterns = [
-    path('', include(router.urls))
+    path('', include(router.urls)),
+    path('upload/artifacts/<str:filename>/', views.UploadArtifactsView.as_view()),
+    path('upload/artifacts/beta/<str:filename>/', views.UploadArtifactsBetaView.as_view()),
```

Comment:
As I wrote should this be in api or in another app?

Reply:
decided to leave in this form as will ad more upload links (for example for menu photos upload etc)